wasmplugin/plugin.go

```diff
@@ -454,13 +460,13 @@ func (ctx *httpContext) OnHttpStreamDone() {
 
 const noGRPCStream int32 = -1
 
-func (ctx *httpContext) handleInterruption(phase string, interruption *ctypes.Interruption) types.Action {
+func (ctx *httpContext) handleInterruption(phase string, interruption *ctypes.Interruption, identifier string) types.Action {
```

Comment:
It looks like we don't need to pass `identifier` since it's on `httpContext`

Reply:
thanks for the input, just realized that handleInterruption in on httpContext